5minutes.html

```diff
@@ -43,6 +43,11 @@ <h1>Projekty</h1>
                     </section>
                             <section>
                                 <img src="img/volontulo.png" style="background:none; border:none; box-shadow:none; "/>
+                                <a href="http://volontuloapp.org/o">http://volontuloapp.org/o</a>
+                            </section>
+                            <section>
+                                <img src="img/streetmix.png" style="background:none; border:none; box-shadow:none; "/>
```

Comment:
if you want to introduce it here, you should remove it from `Przyszłość` part (also threre is already a screenshot there)

Reply:
But I already removed it from `Przyszłość` part, and I change screenshot because now when streetmix is translated it will be nice to show polish version.